tools/esdlcmd/esdlcmd_common.hpp

```diff
@@ -169,10 +321,149 @@ class EsdlCmdHelper : public CInterface
         return new EsdlCmdHelper();
     }
 
-    void loadDefinition(const char * sourceFileName, const char * serviceName, double version, const char* includePath)
+    void loadDefinition(const char * sourceFileName, const char * serviceName, double version, const char* includePath, const char* traceCategory)
     {
         if (!esdlDef.get())
             esdlDef.set(createEsdlDefinition());
+        if (!esdlDef->queryReporter())
+            esdlDef->setReporter(new EsdlCmdReporter());
+        IEsdlDefReporter* reporter = esdlDef->queryReporter();
+        reporter->setFlags(IEsdlDefReporter::Flags(-1), false);
```

Comment:
So by default no debug messages will be reported? Maybe have a comment in code here or in the usage message what the default debug messages output are.

Reply:
This is resetting the reporter flags before processing the command line options. The reset is in case the reporter was created elsewhere and has been initialized.

For anything to be on by default would further complicate the interface because it would be necessary to provide an option to turn those off in addition to the option for turning things on. I think that, if given an option to specify which debug output is desired, no debug output is requested that the obvious expectation is that no debug output will be generated.

Reply:
This is resetting the reporter flags before processing the command line options. The reset is in case the reporter was created elsewhere and has been initialized.

For anything to be on by default would further complicate the interface because it would be necessary to provide an option to turn those off in addition to the option for turning things on. I think that, if given an option to specify which debug output is desired, no debug output is requested that the obvious expectation is that no debug output will be generated.

Furthermore, the output controlled by these flags is new. The flags do not yet affect existing output. This is another reason why it can, and perhaps should, be assumed that nothing will be reported that is not explicitly requested.